packages/web/components/control/token-select-limit.tsx

```diff
@@ -0,0 +1,227 @@
+      () =>
+        !isLoadingQuotePrice && quoteCoinPrice
+          ? new PricePretty(
+              DEFAULT_VS_CURRENCY,
+              quoteCoinPrice.mul(quoteBalance)
+            )
+          : new PricePretty(DEFAULT_VS_CURRENCY, 0),
+      [quoteCoinPrice, quoteBalance, isLoadingQuotePrice]
+    );
+    return (
+      <div>
+        <div className="align-center relative z-10 flex flex-row place-content-between items-center rounded-xl bg-osmoverse-850 py-5 px-3 md:justify-start">
+          {baseAsset && (
+            <div
+              className={classNames(
+                "flex items-center gap-2 text-left transition-opacity",
+                tokenSelectionAvailable ? "cursor-pointer" : "cursor-default",
+                {
+                  "opacity-40": disabled,
+                }
+              )}
+            >
+              {baseAsset.coinImageUrl && (
+                <div className="mr-1 h-[50px] w-[50px] shrink-0 rounded-full md:h-7 md:w-7">
+                  <Image
+                    src={baseAsset.coinImageUrl}
+                    alt="token icon"
+                    width={isMobile ? 30 : 50}
+                    height={isMobile ? 30 : 50}
+                    priority
+                  />
+                </div>
+              )}
+              <div className="flex flex-col">
+                <div className="mr-2 flex items-center">
+                  {isMobile || baseAsset.coinName.length > 6 ? (
+                    <span className="text-h6">{baseAsset.coinName}</span>
+                  ) : (
+                    <h6>{baseAsset.coinName}</h6>
+                  )}
+                  <span className="md:caption ml-2 w-32 truncate text-h6 text-osmoverse-400">
+                    {baseAsset.coinDenom}
+                  </span>
+                </div>
+                {orderDirection === OrderDirection.Ask && (
+                  <>
+                    {!isLoadingBasePrice && baseFiatBalance ? (
+                      <div className="flex text-body1 text-osmoverse-300">
+                        {formatPretty(baseFiatBalance)} available
+                      </div>
+                    ) : (
+                      <div />
+                    )}
+                  </>
+                )}
+              </div>
+            </div>
+          )}
+          {tokenSelectionAvailable && (
+            <button
+              onClick={(e) => {
+                e.stopPropagation();
+                if (tokenSelectionAvailable) {
+                  setIsSelectOpen(!isSelectOpen);
+                }
+              }}
+              className="h-[32px] rounded-2xl bg-osmoverse-800 py-1 px-3 text-body2 text-wosmongton-200"
+            >
+              Change
+            </button>
+          )}
+        </div>
+        <div className="align-center relative z-0 mt-[-20px] flex flex-row place-content-between items-center rounded-xl bg-osmoverse-1000 py-5 px-3 pt-10 md:justify-start">
```

Comment:
```suggestion
        <div className="align-center relative z-0 -mt-5 flex place-content-between items-center rounded-xl bg-osmoverse-1000 py-5 px-3 pt-10 md:justify-start">
```

keep in mind that the default flex direction is always row

Reply:
https://github.com/osmosis-labs/osmosis-frontend/pull/3308/commits/ec8948e28a313965a1ccf4618a135b928c1ac119